### PR TITLE
Fix spelling across the repository

### PR DESCRIPTION
**Pull request by mattpap**
I used [codespell](https://github.com/codespell-project/codespell) for this. This tool could be used to automatically check spelling (including precommit), but this will require some configuration (e.g. a custom dictionary).

### CODE REVIEW

File: bokehjs/src/lib/api/themes.ts [hunk 1]
```diff
@@ -209,7 +209,7 @@ export const caliber = new Theme([
   }),
 ])
 
-export const constrast = new Theme([
+export const contrast = new Theme([
```
Comment (jbednar) [inline bokehjs/src/lib/api/themes.ts:212]: It might be good to make two separate PRs for the changes that affect code and the ones that affect only comments,  docs, and error messages.  That way any potential breakages can be localized to a much smaller set of changes.

Comment (mattpap) [reply bokehjs/src/lib/api/themes.ts:212]: I added a deprecation for the previous name.

Comment (mattpap) [reply bokehjs/src/lib/api/themes.ts:212]: Given this is the only API change, I will keep it in this PR.

**Review by bryevdv — APPROVED** [review]
